LWCE_Core/Src/XComLongWarCommunityEdition/Classes/LWCE_UITacticalHUD_Radar.uc

```diff
@@ -90,4 +90,128 @@ simulated function UpdateActiveUnit()
     }
 
     m_hEnemyArrWatchHandle = WorldInfo.MyWatchVariableMgr.RegisterWatchVariable(m_kActiveUnit, 'm_arrVisibleEnemies', self, UpdateBlips);
-}
\ No newline at end of file
+}
+
+simulated function UpdateBlips()
+{
+    local int I;
+    local XComMeldContainerActor Index;
+    local string Data;
+    local XGUnit tmpUnit, ActiveUnit;
+    local XComInteractiveLevelActor ILA;
+    local float MaxDistance;
+
+    ActiveUnit = XGUnit(m_kActiveUnit);
+    if ((ActiveUnit == none) || XComTacticalController(controllerRef) == none)
+    {
+        return;
+    }
+    m_iCurrUnitsSightRadius = class'XGTacticalGameCore'.default.BASE_REMOVAL_DAYS * 64;
+    I = 0;
+    while (I < m_arrBlips.Length)
```

Comment:
Can you change this (and the other `while` loops below) to a `for` loop please? I think that's what the original was, and it's more idiomatic anyway.